rust/examples/spin-basic/src/lib.rs

```diff
@@ -74,6 +84,24 @@ fn handle_spin_basic(_req: Request) -> anyhow::Result<impl IntoResponse> {
     histogram.record(9, attrs);
     gauge.record(8, attrs);
 
+    // ------------
+    // --- Logs ---
+    // ------------
+
+    // Set up a LoggerProvider using the WASI log processor.
+    let log_resource = Resource::builder().with_service_name("spin-logs").build();
+    let processor = opentelemetry_wasi::WasiLogProcessor::new(Some(log_resource.clone()));
+    let provider = opentelemetry_sdk::logs::SdkLoggerProvider::builder()
+        .with_resource(log_resource)
+        .with_log_processor(processor)
+        .build();
+
+    let layer = layer::OpenTelemetryTracingBridge::new(&provider);
+    tracing_subscriber::registry().with(layer).init();
+
+    // Log something
+    tracing::info!(message = "Traces and metrics and logs, oh my!");
```

Comment:
Perhaps this example should live in our `spin-tracing` example where we already use https://docs.rs/tracing/latest/tracing/?

This isn't a problem to solve for this PR but I'm not sure how to best structure our examples b/c there is going to be a M x N x P sized matrix of possible examples where
- M is language e.g. Rust, JS, etc.
- N is common practice e.g. use opentelemetry SDKs directly or use something language native like [tracing](https://docs.rs/tracing/latest/tracing/) backed by opentelemetry SDKs.
- P is signal e.g. traces, metrics, logs.

Reply:
I forgot another dimension of the matrix: runtime (O). So it's really M x N x P x O. Anyways...

2 per language-runtime pairing?